src/status_im/contexts/wallet/events.cljs

```diff
@@ -185,7 +185,8 @@
                 :navigate-to-account address
                 :new-account?        true)
     :fx [[:dispatch [:wallet/get-accounts]]
-         [:dispatch [:wallet/clear-new-keypair]]]}))
+         [:dispatch [:wallet/clear-new-keypair]]
+         [:dispatch [:wallet/request-collectibles-for-current-viewing-account]]]}))
```

Comment:
Is this not a bit strange since the current viewing account would not even be set?

Reply:
When the account is created, current viewing account is set and then user is navigated there. So it's working properly

Reply:
This solution work for the current viewing account and it's good :+1: , but I don't think it'll work for the wallet home screen.

What I'd do is reset the fetching state of collectibles (yes, we have a state for it, I'll explain later), and request again the collectibles for all accounts when we visit the home screen, it'd be the like re-login.

And don't worry, it's not expensive because the collectible fetching is optimized to request in batches of 25 (if I recall well) evenly divided between the number of accounts. I picked this number because the processing time shouldn't lock the JS thread.

Related to the fetching state, when we are in the wallet home screen, we load the collectibles in batches as we scroll the flat-list, basically, we use a "cursor" to track the next batch, we also discard accounts that said they don't have more collectibles to fetch from the following requests. So if a new account is added, it seems it should be considered from the beginning in this state.

Reply:
> @ulisesmac Everything is working perfectly with current implementation. This event `wallet/request-collectibles-for-current-viewing-account` will update the total balance and everything else in everywhere.
> 
> Please let me know if I still need to change it.


Hey @mmilad75 , yeah, it'll work for a single account, but if you are already watching an account with collectibles and then you add a new account with more collectibles, I'm pretty sure the home screen won't show them uiniformily, and probably while scrolling you'll face some jumps in the listing.

If this is not happening, then all is ok. Could you please check the case I mentioned?
Thank you!

Reply:
I added dispatching `wallet/request-collectibles-for-current-viewing-account` as well.